float32[] x_coords_chair
float32[] y_coords_chair
---
int8 next_state
float32 x_coord_chair
float32 y_coord_chair